src/js/pm-image-editor.js

```diff
@@ -455,6 +474,16 @@
 
                 $scope.updateEditorCss();
             });
+
+            var watchSelectionDimensions = function(value) {
+                if (parseInt(value)) {
```

Comment:
That's not what `parseInt` guarantees though. Look at the examples in this page: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt